Country.java

```diff
@@ -0,0 +1,49 @@
+package firstPackage;
```

Comment:
Please do not include .class file in pull request. Add it in .gitignore.

Reply:
Done !